compose/release-notes.md

```diff
@@ -8,6 +8,38 @@ redirect_from:
 ---
 {% include compose-eol.md %}
 
+## 2.18.0
+{% include release-date.html date="2023-05-16" %}
+
+### Enhancements
+- Dry run support
+- First (alpha) implementation of viz subcommand
+- introduce `--no-path-resolution` to skip relative path to be resolved
+- introduce `COMPOSE_ANSI` to define `--ansi` default value
+- introduce `COMPOSE_STATUS_STDOUT` to get status messages sent to stdout
+
+### Bug fixes
```

Comment:
Are there specific PRs we can add as we do in previous release notes?

Reply:
is this a requirement ? 
This is a synthesis for https://github.com/docker/compose/releases/tag/v2.18.0, which includes individual links

Reply:
It's just how it's always been done for Compose and how we do Desktop. But agree that it is a little silly to repeat all the links. Perhaps we can add something like 'For more detailed information, see [the release notes in the Compose repo](https://github.com/docker/compose/releases/tag/v2.18.0) and add that to line 13?